qa/standalone/mon/osd-erasure-code-profile.sh

```diff
@@ -222,6 +222,17 @@ function TEST_profile_k_sanity() {
         m=1 || return 1
 }
 
+function TEST_invalid_crush_failure_domain() {
+    local dir=$1
+
+    run_mon $dir a || return 1
+
+    local profile=invalid_crush_failure_domain_profile
+    local crush_failure_domain=foo
+
+    ! ceph osd erasure-code-profile set $profile k=4 m=2 crush-failure-domain=$crush_failure_domain 2>&1 || return 1
```

Comment:
@pdvian nit, i think the reason why i was confused is that "foo" is not obvious to me that is does not exist. and "invalid_crush_failure_domain_profile" is very distracting in the sense that i'd believe that the reason why "ceph osd erasure-code-profile set" fails is that the user *choses* an invalid profile. how about
```sh
local profile=ec_profile
local crush_failure_domain=invalid_failure_domain
```